app/build.gradle

```diff
@@ -41,21 +42,26 @@ android {
     }
 
     packagingOptions {
+        exclude 'META-INF/DEPENDENCIES'
         exclude 'META-INF/LICENSE.txt'
         exclude 'META-INF/NOTICE.txt'
     }
-
+    android {
+        lintOptions {
+            abortOnError false
```

Comment:
What does this do? I usually prefer to leave warnings / errors on unless there's a good reason not to.

Reply:
It's because the Android linter is finding 110 issues. By default, the build is failing as long as there are linter issues.
When this option is set to false, the linting task still displays that issues were found and writes a html and xml report, but the build doesn't fail.
If you would like to rather let the build fail until the issues are resolved, I could remove that option from the pull request.